zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -61,6 +61,7 @@
   Date dateUpdated;
   private Map<String, Object> config; // paragraph configs like isOpen, colWidth, etc
   public final GUI settings;          // form and parameter settings
+  private Map<String, Set<String>> runtimeInfos;
```

Comment:
Hello,
I think there is no problem using `transient`.
If you think you have a problem with using `transient`, can you comment on reason?

Actually, my english is not good.
I am sorry if I have been rude to you.
Thank you.